.github/workflows/build-new.yml

```diff
@@ -0,0 +1,76 @@
+name: Pybind11 CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ assigned, opened, synchronize, reopened ]
+  release:
+    types: [published]
+  workflow_dispatch:
+
```

Comment:
@johnwlambert Your repo has an additional trigger `repository_dispatch:    types: [pybind-wrapper]` here, not sure what this means so I omitted it.